server/features/publish.feature

```diff
@@ -0,0 +1,88 @@
+@wip
+Feature: Publish
+
+    @auth
+    Scenario: Publish Event
+        When we post to "/products" with success
+        """
+        {
+            "name":"prod-1","codes":"abc,xyz", "product_type": "both"
```

Comment:
Since we would be publishing event and schema is different. It would be good to write a scenario where event is published to a subscriber and the subscriber is having is content filters matching as well as not matching the event schema field. i.e. product with a category filter and product with a desk filter.